rs/node_rewards/canister/tests/regisry_sync_test.rs

```diff
@@ -1,56 +1,66 @@
 use candid::{Decode, Encode};
 use ic_interfaces_registry::RegistryValue;
-use ic_nns_test_utils::common::{build_node_rewards_test_wasm, NnsInitPayloadsBuilder};
-use ic_nns_test_utils::state_test_helpers::{
-    query, setup_nns_canisters, state_machine_builder_for_nns_tests,
-};
+use ic_nervous_system_integration_tests::pocket_ic_helpers::NnsInstaller;
+use ic_nns_test_utils::common::build_node_rewards_test_wasm;
 use ic_protobuf::registry::subnet::v1::{SubnetRecord, SubnetType};
 use ic_registry_keys::make_subnet_record_key;
 use ic_types::{PrincipalId, SubnetId};
+use pocket_ic::PocketIcBuilder;
 use std::time::Duration;
 
-#[test]
-fn test_registry_value_syncing() {
-    let state_machine = state_machine_builder_for_nns_tests().build();
+#[tokio::test]
+async fn test_registry_value_syncing() {
+    let pocket_ic = PocketIcBuilder::new()
+        .with_nns_subnet()
+        .with_sns_subnet()
```

Comment:
NC

I like this **_MUCH_** better than `for_nns_tests`. In fact, any `for_x` is a red flag for me. The name should tell (me) the behavior, and then the caller (I) can decide whether that behavior is suitable.

There is no such thing as a bolt "for the Space Shuttle". No bolt manufacturer can bid on such a request for quote. There is only such thing as a bolt with G geometry, S strength, H heat resistance, etc. It is then up to the Space Shuttle to decide whether G, S, H, etc are suitable to the fastening task it has in mind.